src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/AppointmentService.kt

```diff
@@ -92,7 +94,7 @@ class AppointmentService(
     return appointmentRepository.save(appointment)
   }
 
-  fun submitSessionFeedback(appointment: Appointment, submitter: AuthUser): Appointment {
+  fun submitSessionFeedback(appointment: Appointment, submitter: AuthUser, appointmentType: AppointmentType): Appointment {
```

Comment:
We don't need to pass in appointmentType here. This class should really be renamed to `SupplierAssessmentAppointmentService`.
Action plan appointments use `ActionPlanSessionsService` rather than this class.

Reply:
The idea was to create a generic appointment service that they will both use. Eventually action plan will move to this.

Reply:
👍